repositories/rule_snoozes.go

```diff
@@ -158,13 +158,21 @@ func (repo *MarbleDbRepository) ListRuleSnoozesForDecision(
 		return nil, err
 	}
 
+	if len(snoozeGroupIds) == 0 {
```

Comment:
fix of a possible bug: no filter would be applied if an empty slice is passed (AFAIK)